app/utilities/schema.py

```diff
@@ -19,15 +19,7 @@
 TEST_SCHEMA_DIR = "test_schemas"
 LANGUAGE_CODES = ("en", "cy", "ga", "eo")
```

Comment:
We now have a language code (eo) which doesn't have a language map. Do we even need to maintain eo/ga going forward?

Reply:
Thanks, can remove 'eo', not sure about 'ga' though.

Reply:
Ok, removed 'eo' now!